transbank/webpay/webpayplus/transaction_commit_response.py

```diff
@@ -0,0 +1,129 @@
+class TransactionCommitResponse:
+    def __init__(self, json_data):
+        self._vci = json_data.get('vci', None)
+        self._amount = json_data.get('amount', None)
+        self._status = json_data.get('status', None)
+        self._buy_order = json_data.get('buy_order', None)
+        self._session_id = json_data.get('session_id', None)
+        self._card_number = json_data.get('card_details').get('card_number', None) \
+            if json_data.get('card_details', None) is not None else None
```

Comment:
Maybe these two lines are easier to read if you prefer simpler code rather than burying and if/else there:
```
card_details = json_data.get('card_details', None)
self._card_number = card_details.get('card_number', None)
```

But more importantly: Why are we changing the structure of the response (a nested field being "popped out" to first level)? Is that something we are doing on all languages or just python?

Reply:
You've got the reason about the more readable code, respect to changing the structure of the response, I'm based in the doc "Integración WebpayV2.2.pdf"

Reply:
This is not being done in other languages as far as I can tell

Reply:
In first place, I need to check if the key 'card_details' exists, what happens if 'card_details' comes empty and you try to apply the 'get' method? It raises an exception, that is the reason because I used conditionals preventing exceptions, I've used the php sdk as the closest example

Reply:
Oops, sorry my code example was wrong. What I meant was:

```
card_details = json_data.get('card_details', {})
self._card_number = card_details.get('card_number', None)
```

The other more important part is that the other SDKs (as @ffiebig mentions) are not "flattening" the response structure. So you shouldn't pull the `card_number` into the response but keep the nested structure in your API.